pallets/did/src/service_endpoints.rs

```diff
@@ -59,48 +59,45 @@ pub struct DidEndpoint<T: Config> {
 impl<T: Config> DidEndpoint<T> {
 	/// Validates a given [DidEndpoint] instance against the constraint
 	/// set in the pallet's [Config].
-	pub(crate) fn validate_against_constraints(&self) -> Result<(), InputError> {
+	pub(crate) fn validate_against_constraints(&self) -> Result<(), Input> {
```

Comment:
If we would use the `errors::Input` path everywhere I would think that this change would make more sense. Seeing only `Input` there makes you think that the input is returned in case of an error.

```suggestion
	pub(crate) fn validate_against_constraints(&self) -> Result<(), errors::Input> {
```

Reply:
done